DevCycle/DVCClient.swift

```diff
@@ -182,9 +182,7 @@ public class DVCClient {
     }
 
     private func handleCachedAnonUserId(user: DVCUser) {
```

Comment:
do we still need this function? 🤔 or can we simply just call `self.cacheService.setAnonUserId(anonUserId: userId)` in the existing places this function is used?

Reply:
I think it's still useful to re-use that condition. I updated to generic `cacheUser` method so that we can also include saving the user object to the cache